content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,296 @@ weight: 50
 keywords: [kubernetes,vms]
 ---
 
-{{< warning_icon >}} Mesh expansion is broken in the initial release of Istio 1.0. We expect to have a patch available that
-fixes this problem very soon. We apologize for the inconvenience.
-
-Instructions for integrating VMs and bare metal hosts into an Istio mesh
+This guide provides instructions for integrating VMs and bare metal hosts into an Istio mesh
 deployed on Kubernetes.
 
 ## Prerequisites
 
-* Setup Istio on Kubernetes by following the instructions in the [Installation guide](/docs/setup/kubernetes/quick-start/).
+* You have already set up Istio on Kubernetes. If you haven't done so already, you can find out how in the [Installation guide](/docs/setup/kubernetes/quick-start/).
```

Comment:
"already" is used twice, doesn't sound great.

Reply:
Well spotted. :)